pacstall

```diff
@@ -596,12 +596,13 @@ Helpful links:
                 fi
                 if ! source "$STGDIR/scripts/install-local.sh"; then
                     fancy_message error "Failed to install ${GREEN}${PACKAGE}${NC}"
-            		rm -rf "${SRCDIR:?}"
+                    if ! [[ -f "/tmp/pacstall-pacdeps-$PACKAGE" ]]; then
+                        rm -rf "${SRCDIR:?}"
+                    fi
                     exit 1
                 fi
                 shift
             done
-            rm -rf "${SRCDIR:?}"
```

Comment:
Keep this, but with the same `pacdep` check

Reply:
Or else `/tmp/pacstall` doesn't get deleted

Reply:
I tried with:
```bash
if ! [[ -f "/tmp/pacstall-pacdeps-$PACKAGE" ]]; then
	sudo rm -rf "${SRCDIR:?}"
fi
```
and it didn't work..